packages/0x.js/scripts/postpublish.js

```diff
@@ -0,0 +1,43 @@
+const execAsync = require('async-child-process').execAsync;
+const postpublish_utils = require('../../../scripts/postpublish_utils');
+const packageJSON = require('../package.json');
+
+const cwd = __dirname + '/..';
+const subPackageName = packageJSON.name;
+
+let tag;
+let version;
+postpublish_utils.getLatestTagAndVersionAsync(subPackageName)
+    .then(function(result) {
+        console.log('POSTPUBLISH: Releasing...');
+        tag = result.tag;
+        version = result.version;
+         const releaseName = subPackageName + ' v' + version;
```

Comment:
we should put this release name formatting into a postpublish_utils function since its repeated across all the scripts